src/pages/api/data-requests.js

```diff
@@ -38,21 +39,24 @@ export default async (req, res) => {
 
   switch (method) {
     case 'POST': {
-      const response = { status: 204, message: '' }
-      // TEMP
-      const githubSuccess = false
-      const hubspotSuccess = false
+      const response = { status: null }
+      const githubSuccess = false // TEMP
+      const hubspotSuccess = await submitHubspotDataRequest(
+        process.env.HUBSPOT_ACCESS_TOKEN,
+        requestValues,
+        requestValues.request_type
+      )
 
       // requests to Slack only if requests to GitHub and/or HubSpot fail
       if (!githubSuccess || !hubspotSuccess) {
-        let failedRequest
-        if (!githubSuccess && !hubspotSuccess) {
-          failedRequest = 'GitHub and HubSpot'
-        } else if (!githubSuccess) {
-          failedRequest = 'GitHub'
-        } else {
-          failedRequest = 'HubSpot'
-        }
+        // sets failed requests' API name(s) to print
+        const failedRequest =
+          // eslint-disable-next-line no-nested-ternary
+          !githubSuccess && !hubspotSuccess
+            ? 'GitHub and HubSpot'
+            : !githubSuccess
+            ? 'GitHub'
+            : 'HubSpot'
```

Comment:
Break this up into multiple lines for clarity.

ex:
```suggestion
        // sets failed requests' API name(s) to print
        const bothFailed = !githubSuccess && !hubspotSuccess
        const oneFailed = !githubSuccess ? "GitHub" : "Hubspot"
        const failedRequest = bothFailed ? "GitHub and Hubspot" : oneFailed
```

Reply:
See the commit at 3880d59ee752189c79ab237c3d8735dd850275b8.